.github/workflows/CI.yml

```diff
@@ -42,7 +42,12 @@ jobs:
         run: julia -e 'using Pkg; Pkg.Registry.add(RegistrySpec(url="https://github.com/JuliaMolSim/MolSim"))'
 
       - uses: julia-actions/julia-buildpkg@v1
+        env:
+          LD_LIBRARY_PATH: $HOME/libxc-5.1.5/lib
```

Comment:
Why is there a libxc dependency here? this only seems to be in the CI. does anything in NQCModels or its tests require that?

Reply:
test/ase_pycall.jl:17-22 uses it to test the ASE calculator interface with GPAW. I couldn't find a working conda package for GPAW, or Id've changed the test mechanism to install it through CondaPkg.jl and make the tests work on any machine without having to set up ase and GPAW first.